### ISSUE TITLE
Failed to restart the db-sync

### ISSUE THREAD

**Issue by saberraz** [lutraconsulting/mergin-db-sync#42]
```
Logging in to Mergin...
Connecting to the database...
Modified and base schemas already exist
Downloading version v8 of Mergin project saber/survey_postgis to /tmp/dbsync
GEODIFF: Error: Cannot create driver host=xps dbname=survey user=postgres password=postgres
Error: Failed to create a copy of base source for driver host=xps dbname=survey user=postgres password=password

Traceback (most recent call last):
  File "dbsync_daemon.py", line 65, in <module>
    main()
  File "dbsync_daemon.py", line 37, in main
    dbsync.dbsync_init(mc, from_gpkg=False)
  File "/mergin-db-sync/dbsync.py", line 608, in dbsync_init
    summary_modified = _compare_datasets(config.db_conn_info, config.db_schema_modified,
  File "/mergin-db-sync/dbsync.py", line 211, in _compare_datasets
    _geodiff_create_changeset_dr(src_driver, src_conn_info, src, dst_driver, dst_conn_info, dst, tmp_changeset)
  File "/mergin-db-sync/dbsync.py", line 203, in _geodiff_create_changeset_dr
    _run_geodiff([config.geodiffinfo_exe, "createChangesetDr", src_driver, src_conn_info, src, dst_driver, dst_conn_info, dst, changeset])
  File "/mergin-db-sync/dbsync.py", line 153, in _run_geodiff
    raise DbSyncError("geodiffinfo failed!\n" + str(cmd))
dbsync.DbSyncError: geodiffinfo failed!
['/geodiff/build/geodiffinfo', 'createChangesetDr', 'host=xps dbname=survey user=postgres password=password', 'photos', 'sqlite', '', '/tmp/dbsync/photos.gpkg', 'postgres', '/tmp/tukTXjIf']
```

**Comment by saberraz** [comment]
https://public.cloudmergin.com/projects/saber/survey_postgis/ is the project. And here is the database dump:
[survey.zip](https://github.com/lutraconsulting/mergin-db-sync/files/6352719/survey.zip)
(change zip extension to backup!)

**Comment by saberraz** [comment]
The script:
`docker run --name mergin_db_sync -it   -e DB_CONN_INFO="host=xps dbname=survey user=postgres password=password"   -e DB_SCHEMA_MODIFIED=photos   -e DB_SCHEMA_BASE=photos_base   -e MERGIN_USERNAME=saber   -e MERGIN_PASSWORD=password   -e MERGIN_PROJECT_NAME=saber/survey_postgis   -e MERGIN_SYNC_FILE=photos.gpkg   lutraconsulting/mergin-db-sync:latest python3 dbsync_daemon.py --init-from-db`

**Comment by wonder-sk** [comment]
Fixed in fa4c793
